agent/custom/bbc_action.py

```
@@ -132,7 +132,12 @@ def on_popup(msg):
                 popup_message = msg.get('popup_message', '')
                 mfaalog.info(f"[ExecuteBbcTask] 收到弹窗: {popup_title} - {popup_message}")
                 if not state['finished']:
-                    self._handle_popups([msg], support_order_mismatch, team_config_error, state, manager)
+                    try:
+                        # 尝试获取 manager 实例
+                        local_manager = get_manager()
+                        self._handle_popups([msg], support_order_mismatch, team_config_error, state, local_manager)
+                    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)